### PR TITLE
Merge fix: "use wget instead of curl #229"

### PR DESCRIPTION
**Pull request by nicolae06**
Closes https://github.com/nf-core/fetchngs/issues/169

### CODE REVIEW

**Comment by nicolae06** [comment]
Hi  @drpatelh , I tried to redo PR #229 and fix the merge. Could you take a look? Thank you!

**Comment by maxulysse** [comment]
I see no reasons not to switch from curl to wget

**Comment by maxulysse** [comment]
@nicolae06 I updated the CHANGELOG, so you'll need to git pull before doing any changes.

Since you changed a process, the corresponding tests are now failing.
Can you try to run nf-test with the following command:

`nf-test test --tag sra_fastq_ftp --profile "test,docker" --updateSnapshot` and commit the newly changed snapshot file.

**Comment by nicolae06** [comment]
Hi @maxulysse, I executed your command, but I removed the arguments as I encountered errors while attempting to run the test. The errors indicated that '--retry 5 --continue-at - --max-time 1200' are not recognized by wget. I checked the 'man wget' command, but these flags were not listed there.

**Comment by suhrig** [comment]
Thanks @nicolae06 for taking care of this! And apologies that I didn't manage.

File: conf/modules.config [hunk 1]
```diff
@@ -0,0 +1,171 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
```
Comment (drpatelh) [inline conf/modules.config:3]: This file is no longer needed in the current version of the pipeline so needs to be deleted.

**Comment by drpatelh** [comment]
Thanks @nicolae06 ! The failing tests should be fixed when we merge in the template updates in https://github.com/nf-core/fetchngs/pull/242

**Review by drpatelh — APPROVED** [review]
🚀

**Pull request merged**